.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
```

Comment:
I see there are additional settings we can add such as the following:

  schedule:
    interval: daily
    time: "13:00"
  open-pull-requests-limit: 10

Should we set a PR limit?  I fear this could result in just too much noise for us to deal with. Also we should probably set the "time" value to a time when out VMs are online.  They are offline for deployment from 1:00 AM - ~5:00 AM PST every day and we don't want PRs getting created then because the CI jobs will fail if the VMs are unavailable